.template/addons/bootstrap/application.js.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+insert_into_file 'app/javascript/packs/application.js', after: %r{import 'core-js/stable'.+\n} do
+  <<~JAVASCRIPT
+    import 'vendor/bootstrap';
```

Comment:
Can we just `require('bootstrap.native')` instead of create our own boostrap native js file and import it?
What do you think?

Reply:
The conventions that we follow in our projects is that we don't directly require third-party modules to the main file, in this case, `application.js`. We create a folder with respect to the name of the third-party plugin in `vendor` and import them in the main file. This gives us a certain level of independence if we want to override anything from the third-party and thus the `application.js` stays clean.

cc. @andyduong1920 is my explanation okay?

Reply:
First of all, we never override the third-party directly @rafayet-monon :D 

And the reason we put the bootstrap in vendor was it comes from a previous project, I don't set it up but me main use case is it's easy to look on that source code in order to customize something on my side.

Another thing, as the SCSS isn't handle by Webpakc in Rails, it's handle by Sprocket so I think at that time, people choose the Rails way, putting the scss to vendor folder or using the boostrap gem,

So it's a good time to test that again, @rafayet-monon could you spend time to try another solution

- Manage boostrap package with `package.json`
- Import the boostrap.scss from in the application.scss from the `node_modules/boostrap/` 

cc @gutakk